src/puppetlabs/puppetdb/anonymizer.clj

```diff
@@ -449,6 +449,13 @@
     (-> wire-node
         (update "certname" anonymize-leaf :node context config))))
 
+(defn anonymize-catalog-inputs
+  "Anonymize a set of catalog inputs"
+  [config wire-catalog-inputs]
+  (let [context {"node" (get wire-catalog-inputs "certname")}]
+    (-> wire-catalog-inputs
+        (update "certname" anonymize-leaf :node context config))))
```

Comment:
Just a note that Zak and I think this might need additional anonymization (particularly for input names) before we can finish up here.